envi/archs/thumb16/disasm.py

```diff
@@ -172,8 +172,13 @@ def rm_reglist(va, value):
     oper0.oflags |= OF_W
     return oper0,oper1
 
-def reglist(va, value):
-    reglist = (value & 0xff) | ((value & 0x100)<<5)
+def pop_reglist(va, value):
```

Comment:
bugfix: split these up.  for push, we're pushing LR in "special case", but in pop, we're popping PC.